### PR TITLE
register new cache in CHTC-ITB (SOFTWARE-5211)

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

File: topology/University of Wisconsin/CHTC/CHTC-ITB.yaml [hunk 1]
```diff
@@ -324,3 +324,38 @@ Resources:
     AllowedVOs:
       - GLOW
       - OSG
+
+  CHTC-ITB-HELP-CACHE1:
```
Comment (matyasselmeci) [inline topology/University of Wisconsin/CHTC/CHTC-ITB.yaml:328]:
```suggestion
  CHTC-ITB-HELM-CACHE1:
```

File: topology/University of Wisconsin/CHTC/CHTC-ITB.yaml [hunk 2]
```diff
@@ -324,3 +324,38 @@ Resources:
...
+          ID: OSG1000002
+          Name: Matyas Selmeci
+      Security Contact:
+        Primary:
+          ID: OSG1000002
+          Name: Matyas Selmeci
+        Secondary:
+          ID: OSG1000015
+          Name: Aaron Moate
+    FQDN: stash-cache.osg.chtc.io
```
Comment (matyasselmeci) [inline topology/University of Wisconsin/CHTC/CHTC-ITB.yaml:352]:
```suggestion
    FQDN: helm-cache.osg.chtc.io
```

Comment (matyasselmeci) [inline topology/University of Wisconsin/CHTC/CHTC-ITB.yaml:352]:
```suggestion
    FQDN: helm-cache1.osgdev.chtc.io
```

File: topology/University of Wisconsin/CHTC/CHTC-ITB.yaml (hunk 1 above)
Comment (edquist) [reply topology/University of Wisconsin/CHTC/CHTC-ITB.yaml:328]: Very nice!

**Review by matyasselmeci — APPROVED** [review]
Fine except for a few things to fix -- see suggestions. Pre-approving.

File: topology/University of Wisconsin/CHTC/CHTC-ITB.yaml (hunk 2 above)
Comment (edquist) [reply topology/University of Wisconsin/CHTC/CHTC-ITB.yaml:352]: derp, missed a spot

**Pull request merged**